js task 1/home.js

```diff
@@ -0,0 +1,14 @@
+const str ='develop';
+function slicee(str){
```

Comment:
The name slicee is a bit confusing, as it duplicates the built-in slice() method and doesn’t give a clear understanding of its role. A better name could be getLastThreeChars() to make the function’s purpose clearer